src/annotator/anchoring/types.js

```diff
@@ -31,16 +24,16 @@ import { matchQuote } from './match-quote';
 export class RangeAnchor {
   /**
    * @param {Node} root - A root element from which to anchor.
-   * @param {AnyRangeType} range -  A range describing the anchor.
+   * @param {Range} range -  A range describing the anchor.
    */
   constructor(root, range) {
     this.root = root;
-    this.range = sniff(range).normalize(this.root);
+    this.range = range;
```

Comment:
This is the change that prevents anchoring of `RangeSelector` selectors from modifying the DOM.